lib/SPIRV/VectorComputeUtil.h

```diff
@@ -0,0 +1,131 @@
+//=- VectorComputeUtil.h - vector compute utilities declarations -*- C++ -*-=//
+//
+//                     The LLVM/SPIR-V Translator
```

Comment:
Please add the license header as in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/lib/SPIRV/LLVMToSPIRVDbgTran.cpp

Reply:
Fixed